DigitalLearningSolutions.Data/Models/Centres/CentreSummaryForFindYourCentre.cs

```diff
@@ -29,5 +29,19 @@ public override string SearchableName
             nameof(RegionName),
             RegionName
         );
+
+        public string WebsiteHref => GenerateWebsiteHref();
+
+        public string EmailHref => GenerateEmailHref();
+
+        private string GenerateWebsiteHref()
+        {
+            return $"https://{WebUrl}";
+        }
+
+        private string GenerateEmailHref()
+        {
+            return $"mailto:{Email}";
+        }

```

Comment:
I don't think these need the methods as well as the properties, just WebsiteHref => $"https:{url}"
Also is it possible we need to make this smarter? What happens if the url has been entered with https:// already?